File: AirportManagement.API/AirportManagement.API/Data/Models/BaggageCheck.cs
using System;
namespace AirportManagement.API.Models
{
	public class BaggageCheck
	{
		public int Id { get; set; }

		public string CheckResult { get; set; }

		public DateTime CreatedAt { get; set; }

		public DateTime UpdatedAt { get; set; }

		public int BookingId { get; set; }

		public Booking? Booking { get; set; }

		public int PassangerId { get; set; }

		public Passangers? Passangers { get; set; }
	}
}

File: AirportManagement.SPA/src/app/_models/BaggageCheck.ts
import { Booking } from "./Booking";
import { Passangers } from "./Passangers";

export interface BaggageCheck {
    Id: number;
    CheckResult: string;
    CreatedAt: Date;
    UpdatedAt: Date;
    BookingId: number;
    Booking: Booking;
    PassangerId: number;
    Passangers: Passangers;
}
